app/presenters/manual_presenter.rb

```diff
@@ -1,7 +1,13 @@
 require "govspeak_to_html_renderer"
 
 class ManualPresenter
-  def call(manual)
+  attr_reader :manual
```

Comment:
Minor/subjective: Does this reader method need to be public or do we even need it at all?